test/smartwallet/SmartWallet.test.ts

```diff
@@ -1,90 +0,0 @@
-import { BaseProvider } from '@ethersproject/providers';
```

Comment:
why did we delete this file? are we covering this tests in other file?

Reply:
I just changed the name to lowercase.

Reply:
I was not able to find it. Maybe you haven't include the file in git.

Reply:
Yes, it's because I also changed the name. It's rifSmartWallet.test.ts, sorry.

Reply:
Yes, it's because I also changed the name. It's rifSmartWallet.test.ts, sorry.

Reply:
but still cannot find it

Reply:
haha, we changed the name once again. It's this one: https://github.com/rsksmart/rif-relay/blob/c76689a3c557e0ea184fb8badc76679d3c14be81/test/smartwallet/utils.ts